Makefile

```diff
@@ -1,7 +1,7 @@
 VERSION=0.9.16
 SOURCE?=./...
 VINYLDNS_REPO=github.com/vinyldns/vinyldns
-VINYLDNS_VERSION=0.9.10
+VINYLDNS_VERSION=0.20.0
```

Comment:
We are trying to update Vinyldns api version to latest in all the clients. If the same changes added here, it would be appreciated.

Reply:
Hi @Jay07GIT  I did set this to 0.20.0 already which is the latest version of Vinyldns here https://github.com/vinyldns/vinyldns/releases

or am I misunderstanding your ask?

Reply:
@Jay07GIT  - just pushed a commit where I changed it to ensure it's always pulling the current from master branch of vinyldns before doing integration tests. I don't want to make changes here to switch to using docker image directly as that would require including dockerfile & bind configs in this repo then and I'm trying to keep this specific PR to just updating things with same functionality. Happy to improve in future PRs though.

Reply:
@Jay07GIT  - just updated it so it defaults to latest but you can easily change the variable in the makefile if you want to test a specific version for testing